aws/lambda/log-classifier/ruleset.toml

```diff
@@ -231,3 +231,11 @@ pattern = 'no space left on device'
 [[rule]]
 name = 'GHA error'
 pattern = '^##\[error\](.*)'
+
+[[rule]]
```

Comment:
You should move this before `GHA error` rule as matching rule is found top to bottom.  Putting them here would be a no-op as the `GHA error` would still match first

Reply:
oops! thanks. will update